src/main/resources/application.properties

```diff
@@ -76,7 +76,7 @@ quarkus.datasource.min-size=3
 quarkus.datasource.max-size=13
 quarkus.hibernate-orm.database.generation=drop-and-create
 
-%dev.quarkus.datasource.url=jdbc:h2:mem:snowdrop-bot;DB_CLOSE_DELAY=-1;DB_CLOSE_DELAY=-1;INIT=RUNSCRIPT FROM 'classpath:import.sql'
+%dev.quarkus.datasource.url=jdbc:h2:mem:snowdrop-bot;DB_CLOSE_DELAY=-1;DB_CLOSE_DELAY=-1
```

Comment:
The `DB_CLOSE_DELAY` is defined `twice`. To be fixed ;-)

Reply:
If the import.sql script is not injected, how will the DB table created and user admin/admin created ?